website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
 | [sys/config/auditing/request-headers](/vault/api-docs/system/config-auditing)                                                                          | GET               | List the request headers that are configured to be audited                                                                                          |
-| [sys/config/auditing/request-headers:name](/vault/api-docs/system/config-auditing)                                                                     | GET, PUT, DELETE  | Manage the auditing headers (create, update, read and delete)                                                                                       |
-| [sys/config/cors](/vault/api-docs/system/config-cors)                                                                                                  | GET, PUT, DELETE  | Configure CORS setting                                                                                                                              |
-| [sys/config-ui](/vault/api-docs/system/config-ui)                                                                                                      | GET               | Configure the UI settings                                                                                                                           |
-| [sys/internal/specs/openapi](/vault/api-docs/system/internal-specs-openapi)                                                                           | GET               | Generate an OpenAPI document of the mounted backends                                                                                                |
+| [sys/config/auditing/request-headers/:name](/vault/api-docs/system/config-auditing)                                                                    | GET, POST, DELETE | Manage the auditing headers (create, update, read and delete)                                                                                       |
+| [sys/config/cors](/vault/api-docs/system/config-cors)                                                                                                  | GET, POST, DELETE | Configure CORS setting                                                                                                                              |
+| [sys/config-ui/headers](/vault/api-docs/system/config-ui)                                                                                              | GET               | Configure the UI settings                                                                                                                           |
+| [sys/config-ui/headers/:name](/vault/api-docs/system/config-ui#name)                                                                                   | POST, DELETE      | Configure the values to be returned for the UI header                                                                                               |
```

Comment:
Path: Should be `sys/config/ui/headers/:name` - slash not dash

Verbs: Also supports GET

Description: e.g. `Configure custom HTTP headers to be served with the UI`

Reply:
Resolved in upcoming commit.